Feature/modeling and vis and tracking (#1)
snowplow/consent-accelerator: content/tracking/tracking_2.md

+++
title = "Connect Snowplow to your CMP events"
weight = 2
+++

> To manage consent selection for your website visitors, most companies prefer to use products called consent management platforms. By using one of these products on your site, your users can read about the ways and purposes data are collected from your website and then accept, deny or configure which scopes they consent to and which they do not.
> 

Reviewer: ```suggestion
Most companies use consent management platforms to manage consent selection for their website visitors. A CMP allows users to understand how and why data is collected from a website and then accept, deny or configure which scopes they consent to.

```

> There are many consent management platforms out there, which are excellent, and for this demonstration we are going to use [Cookiebot](https://www.cookiebot.com/). We are going to showcase how you can use Cookiebot methods to hook into consent preference selection programmatically and then send these data to Snowplow. Everything shown in this guide can be adjusted for any other consent management platform.

Reviewer: ```suggestion
We will demonstrate how you can use [Cookiebot](https://www.cookiebot.com/) methods to hook into consent preference selection programmatically and send these data to Snowplow. Most CMPs have similar functionality so everything shown in this guide can be adjusted to suit your needs.
```

> <img src="../images/cookiebot.png" alt="Cookiebot" style="border-radius: 8px;" /> 

Reviewer: ```suggestion
![Cookiebot]./images/cookiebot.png?width=70pc )
```
70% Might not be the right size but think it needs to be slightly smaller than it is currently
